src/app/components/team/TeamBots.js

```diff
@@ -188,157 +189,160 @@ class TeamBotsComponent extends Component {
     return (
       <ContentColumn style={{ maxWidth: 900 }}>
         { team.team_bot_installations.edges.length === 0 ?
-          <p style={{ paddingBottom: units(5), textAlign: 'center' }}>
+          <Box pb={5} display='flex' alignItems='center' justifyContent='center'>
```

Comment:
Please use double quotes for string props/attributes.

`192:31  error  Unexpected usage of singlequote  jsx-quotes`